src/ReefGuideAPI.jl

```diff
@@ -136,8 +136,16 @@ function setup_regional_data(config::Dict)
         @debug "Using previously generated regional data store."
     elseif isfile(reg_cache_fn)
         @debug "Loading regional data cache from disk"
-        @eval const REGIONAL_DATA = deserialize($(reg_cache_fn))
-    else
+        # updates to packages like DiskArrays can break deserialization
+        try
+            @eval const REGIONAL_DATA = deserialize($(reg_cache_fn))
+        catch err
+            @warn "Failed to deserialize $(reg_cache_fn)" err
+            # cache file will be replaced by initialize_regional_data_cache
```

Comment:
@PeterBaker0 do you think delete cache file here would be better?

Reply:
I think best to delete...